### ISSUE TITLE
DB/Gossip: Alchemist Arbington and High Priestess MacDonnell - Mission Accomplished!

### ISSUE THREAD

**Issue by ZenoX92** [TrinityCore/TrinityCore#20522]
**Description:**

Alchemist Arbington and High Priestess MacDonnell's gossip texts should change after quest "Mission Accomplished! (Alliance)" is finished.

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

TrinityCore rev. 9fa6885 2017-08-08 11:36:46 -0300 (3.3.5 branch) (Win64, Release, Static) (worldserver-daemon)

**TDB version:**  335.63

**Operating system:** Windows 7

**Suggested Fix:**
```sql
-- Alchemist Arbington and High Priestess MacDonnell --> Fix gossip texts
DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`= 14 AND `SourceGroup` IN (3228, 3184);
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(14,3228,3985,0,0,8,0,5238,0,0,0,0,0,'',"Show Gossip Menu Text 3985 if Quest \'Mission Accomplished!\' is rewarded"),
(14,3184,4039,0,0,8,0,5238,0,0,0,0,0,'',"Show Gossip Menu Text 4039 if Quest \'Mission Accomplished!\' is rewarded");

**Comment by tkrokli** [comment]
Please merge the 2 queries, just like I suggested for https://github.com/TrinityCore/TrinityCore/issues/20523#issuecomment-333673747

**Comment by tkrokli** [comment]
Funny, this gives me flashbacks to when I updated the quest gossip script for Dithers & Arbington ;-)

**Comment by ZenoX92** [comment]
@Wyreth posted a better fix for this here #20540.
